derrida/books/views.py

```diff
@@ -735,10 +755,15 @@ def head(self, request, *args, **kwargs):
         '''
         Proxy HTTP headers for image.
         '''
+        # first check if we need to redirect
+        redirect = self._redirect_info(request)
+        if redirect:
+            return redirect
+
         url = self.get_proxy_url(*args, **kwargs)
         remote_response = requests.head(url)
         response = HttpResponse()
-        for header, value in remote_response.headers.iteritems():
+        for header, value in remote_response.headers.items():
```

Comment:
oh interesting https://www.geeksforgeeks.org/difference-between-dict-items-and-dict-iteritems-in-python/

Reply:
I know this because I wrote a lot of python 2.6/2.7 code with `iteritems` and had to adjust! I don't know how this made it into this codebase (probably copied from something else); doesn't seem like it could have ever worked.